packages/oui-datagrid/src/datagrid.directive.js

```diff
@@ -19,7 +19,9 @@ export default () => ({
         onRowSelect: "&",
         onPageChange: "&",
         onCriteriaChanged: "&onCriteriaChange",
-        onSortChange: "&"
+        onSortChange: "&",
+        refreshable: "@?",
+        refreshIconAreaLabel: "@?"
```

Comment:
I think there is no need of an attribute for this. This should be only set in the provider, like you did: eg. https://github.com/ovh-ux/ovh-ui-kit/blob/master/packages/components/password/src/js/password.provider.js#L7

To keep a consistency with the others `aria` translations, it could be clearer to be renamed `ariaResfreshButtonLabel` or `ariaRefreshLabel` (see Password examples). What do you think ?

Reply:
makes sense, we should maintain the consistency